test/plsql.test.ts

```diff
@@ -116,11 +116,30 @@ describe('PlSqlFormatter', () => {
     `);
   });
 
-  it('formats simple SELECT with national characters', () => {
-    const result = format("SELECT N'value'");
+  // TODO: improve formatting of custom functions
+  it('formats SELECT query with CROSS APPLY', () => {
+    const result = format('SELECT a, b FROM t CROSS APPLY fn(t.id)');
     expect(result).toBe(dedent`
       SELECT
-        N'value'
+        a,
+        b
+      FROM
+        t
+      CROSS APPLY
+      fn (t.id)
+    `);
+  });
+
+  it('formats SELECT query with OUTER APPLY', () => {
+    const result = format('SELECT a, b FROM t OUTER APPLY fn(t.id)');
+    expect(result).toBe(dedent`
+      SELECT
+        a,
+        b
+      FROM
+        t
+      OUTER APPLY
+      fn (t.id)
```

Comment:
I think something has gone wrong with the merge in here.

I moved these two tests inside `supportsJoin()` helper.